source/api/presentation/3.0/index.md

```diff
@@ -592,8 +583,44 @@ The value _MUST_ be an array of JSON objects. Each item _MUST_ have the `id`, `t
 }
 ```
 
+##### rights
+
+A link to an external resource that describes the license or rights statement under which this resource may be used. The rationale for the value being a URI and not a human readable text is that typically there is one statement or license for many resources, and the text is too long to be displayed to the user at the same time as the object. If displaying rights information directly to the user is a requirement, or a publisher defined label is needed, then it is _RECOMMENDED_ to include the information using the `requiredStatement` property.
+
+The value _MUST_ be an array of JSON objects, each of which _MUST_ have an `id` and _SHOULD_ have at least one of `type` and `format`.
```

Comment:
This version of `rights`, as opposed to what we had in 2.x as `license`, seems a possible awkward match with the sort of machine readable statement we had imagined (e.g. CC) which likely does support conneg and this isn't well described with e media type etc.

Reply:
I agree ... see the issue on rights. I didn't change any of the text from current prezi3 branch, just moved it.

Reply:
--> https://github.com/IIIF/api/issues/1479